submitted_models/marble_spot_sensor_config_1/launch/vehicle_topics.launch

```diff
@@ -0,0 +1,88 @@
+<launch>
+    <arg name="world_name"/>
+    <arg name="name"/>
+    <arg name="link_prefix" value="/world/$(arg world_name)/model/$(arg name)/link"/>
+    <arg name="joint_prefix" value="/model/$(arg name)/joint"/>
+    <arg name="breadcrumbs" default="0"/>
+    <arg name="has_omnicam_vras" default="0"/>
+    <arg name="has_ouster_lidar" default="1"/>
+    <arg name="has_flir_cameras" default="1" />
+    <arg name="description_print_command" default="$(find ctu_cras_norlab_spot_sensor_config_1)/scripts/print_robot_urdf" />
```

Comment:
`marble_spot_sensor_config_1`

Reply:
Fixed in https://github.com/osrf/subt/commit/7349e416ef7ccac9a4c3775ecf3057d6e17eef8d